src/controller/python/chip/clusters/Attribute.py

```diff
@@ -61,6 +88,75 @@ class AttributeReadResult(AttributeStatus):
     Data: Any = None
 
 
+_AttributeIndex = {}
+
+
+def _BuildAttributeIndex():
+    ''' Locates the right generated cluster object given a set of parameters.
+
+        isClientSideCommand: True if it is a client-to-server command, else False.
+        path: A CommandPath that describes the endpoint, cluster and ID of the command.
+
+        Returns the type of the cluster object if one is found. Otherwise, returns None.
+    '''
+    for clusterName, obj in inspect.getmembers(sys.modules['chip.clusters.Objects']):
```

Comment:
Can we commonize this logic with that for commands in `Command.py`?

Reply:
We cannot because ths is too slow due to tons os attributes.
It takes ~300ms for getting the attribute, which is unacceptable in the case of wildcard read.

Reply:
I think you mis-understood - I'm suggesting that since we have similar logic here to that in `FindCommandClusterObject`, I'm suggesting making a single function that can potentially do both, given some input args.